src/code_759b0_len_61b0.c

```diff
@@ -54,27 +71,88 @@ INCLUDE_ASM(s32, "code_759b0_len_61b0", check_input_use_partner);
 
 INCLUDE_ASM(s32, "code_759b0_len_61b0", func_800DFAAC);
 
-INCLUDE_ASM(s32, "code_759b0_len_61b0", func_800DFBE8);
+void func_800DFBE8(void) {
+    func_800E5A2C();
+    collision_main_lateral();
+    func_800E4BB8();
+
+    if (!(gPlayerStatusPtr->flags & 0x4000000)) {
+        Camera* camera = &gCameras[0];
+
+        camera->targetPos.x = gPlayerStatusPtr->position.x;
+        camera->targetPos.y = gPlayerStatusPtr->position.y;
+        camera->targetPos.z = gPlayerStatusPtr->position.z;
+    }
+}
 
 void clear_player_status(void) {
     mem_clear(&gPlayerStatus, sizeof(gPlayerStatus));
 }
 
-INCLUDE_ASM(s32, "code_759b0_len_61b0", func_800DFC74);
+void func_800DFC74(void) {
+    PlayerStatus* playerStatus = PLAYER_STATUS;
+
+    mem_clear(playerStatus, sizeof(PlayerStatus));
```

Comment:
tangent, but should this mem_clear function maybe be renamed to bzero?

Reply:
I'll add a comment but would rather refrain from renaming things until we know more stuff